packages/astro-rss/README.md

```diff
@@ -202,6 +185,112 @@ export const get = () => rss({
 });
 ```
 
+## `RSSFeedItem`
```

Comment:
I really like how well you've restructured the docs and how well you explained this